packages/specs/protocol/components/accounts.md

```diff
@@ -0,0 +1,19 @@
+# Accounts
+
+This document is a WIP. It is kept here to maintain a consistent structure mirroring the [implementation directory](../../../contracts/contracts/optimistic-ethereum/OVM/).
```

Comment:
I hope to complete this document thoroughly in the near future, but would like to include it as a placeholder. 
If it's a blocker, I can remove it.